### ISSUE TITLE
Are meetups scraped only once a month?

### ISSUE THREAD

**Issue by liesnikov** [papers-we-love/papers-we-love.github.io#136]
**Edit**: oh, [this comment](https://github.com/papers-we-love/papers-we-love/issues/310#issuecomment-181154820) states it's actually once in a 6 hours, so there is indeed a problem.

Basically the title. Or, maybe, there is some lag between adding a page and a meetup to be scraped?

Pull adding the meetup adapter to our page was merged approx. a day ago. #135 
The link seems to be right, at least copy-pasting it into a browser does lead to an meetup page.
https://github.com/papers-we-love/papers-we-love.github.io/blob/aec6c1f1bd2e28fdde30e1b2c9350a936177e69e/data/chapters.json#L291
Event page - https://www.meetup.com/Papers-We-Love-Kyiv/events/248836089/

Screenshot:
http://paperswelove.org/chapter/kyiv/ - ![Screenshot of a chapter page](https://user-images.githubusercontent.com/17168103/37684843-05e6c7dc-2c9a-11e8-8f46-3d64e7d8de4e.png)

**Comment by DarrenN** [comment]
Yeah I noticed that earlier. The JSON from Meetup.com is populating correctly but the HTML isn't rendering. Only think I can think of off hand is the JSON parser doesn't like something in the file (such as Unicode.) Will take some time to figure out.

**Comment by liesnikov** [comment]
Thanks for the prompt reaction.
I wrote it before, it isn't critical for us, as we don't rely on it at all (but would be nice to have nevertheless), so there is no hurry.
Regarding unicode – there is indeed a couple of non-ascii chars in a description, like an emoji, bullet points and n-dashes.

**Comment by DarrenN** [comment]
Ah, its because the date isn't until April. We specifically don't add the meetups until they're within the month as some of the chapters were creating empty meetups for all 12 months of the year in advance. It will show up on April 1st.
